lighthouse-core/test/config/__snapshots__/config-plugin-test.js.snap

```diff
@@ -0,0 +1,28 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ConfigPlugin accepts a well formed plugin 1`] = `
+Object {
+  "audits": Array [
+    Object {
+      "path": "not/a/path/audit.js",
+    },
+  ],
+  "categories": Object {
+    "lighthouse-plugin-nice-plugin": Object {
+      "auditRefs": Array [
+        Object {
+          "id": "nice-audit",
+          "weight": 1,
+        },
+        Object {
+          "id": "viewport",
```

Comment:
> can you not use viewport? :D

haha, I thought the audit was still ok to use. Switched to `installable-manifest` since that'll probably stick around for a while :P